vanilla/invalid/java/org/spongepowered/server/SpongeVanilla.java

```diff
@@ -179,9 +174,17 @@ public void onServerStopping() {
     }
 
     public void onServerStopped() throws IOException {
+<<<<<<< HEAD
         SpongeCommon.postState(GameState.SERVER_STOPPED, SpongeEventFactory.createGameStoppedServerEvent(Sponge.getCauseStackManager().getCurrentCause()));
         ((SqlServiceImpl) this.game.getServiceManager().provideUnchecked(SqlService.class)).close();
         SpongeCommon.getConfigSaveManager().flush();
+=======
+        SpongeImpl.postState(GameState.SERVER_STOPPED, SpongeEventFactory.createGameStoppedServerEvent(Sponge.getCauseStackManager().getCurrentCause()));
+        if (this.game.getServiceProvider().sqlService() instanceof SqlServiceImpl) {
+            ((SqlServiceImpl) this.game.getServiceProvider().sqlService()).close();
+        }
+        SpongeImpl.getConfigSaveManager().flush();
+>>>>>>> 1762dc36f... Add entry point to service provider, make changes to support this
```

Comment:
mmmh

Reply:
At least it's invalid, though I thought I'd picked these all up.